source/agora/test/EnrollmentManager.d

```diff
@@ -237,7 +237,7 @@ unittest
         }
 
         /// see base class
-        public override void createNewNode (Config conf, string file, int line)
+        public override RemoteAPI!TestAPI createNewNode (Config conf, string file, int line)
```

Comment:
Why this change ? It doesn't seem to be used.

Reply:
I used to use the return value at some point, but during refactoring the calling code was removed. I will revert back.